src/ContaoCoreBundle.php

```diff
@@ -47,10 +46,8 @@ public function boot()
      */
     public function build(ContainerBuilder $container)
     {
-        $container->addCompilerPass(new AddContaoResourcesPass($this->getPath() . '/Resources/contao'));
+        parent::build($container);
```

Comment:
It is always a good idea to call the parent implementation as long as it is not abstract unless one intentionally wants to replace the functionality. I am with @aschempp on this one.